modular_skyrat/modules/departmentization/quartermaster.dm

```diff
@@ -0,0 +1,16 @@
+/datum/outfit/job/quartermaster
```

Comment:
Could you cut the overrides down to what needs to be changed instead of replicating the same var w/ the same value?

Reply:
ok